native/mod_manager/balancer.c

```diff
@@ -191,21 +193,35 @@ apr_status_t remove_balancer(mem_t *s, balancerinfo_t *balancer)
  * get the ids for the used (not free) balancers in the table
  * @param pointer to the shared table.
  * @param ids array of int to store the used id (must be big enough).
- * @return number of balancer existing or -1 if error.
+ * @return number of balancer existing or 0.
  */
```

Comment:
This function returns `APR_SUCCESS` always.

Reply:
The function `loc_get_id` should be above the comment as it is for the `get_ids_used_balancer`.

Reply:
Actually the same occurs for loc_read_balancer()